web/themes/custom/subtheme_videogame/src/sass/base/_variables.scss

```diff
@@ -39,8 +39,8 @@ $indigo:  #6610f2 !default;
 $purple:  #6f42c1 !default;
 $pink:    #e83e8c !default;
 $red:     #dc3545 !default;
-$orange:  #F57C00 !default;
-$yellow:  #ffc107 !default;
+$orange:  #F57C00;
```

Comment:
As you have already created a `_colors.scss` file, you can move every color definition to that file to have every color in a single place.